Bundle/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -43,7 +43,11 @@ public function getConfigTreeBuilder()
             ->end()
             ->children()
                 ->variableNode('base_paths')
-                    ->defaultValue(['%kernel.root_dir%/../src', '%kernel.root_dir%/../vendor/victoire/victoire/Bundle/BlogBundle', '%kernel.root_dir%/../vendor/friendsofvictoire'])
+                    ->defaultValue([
+                        '%kernel.root_dir%/../src',
+                        '%kernel.root_dir%/../vendor/victoire',
+                        '%kernel.root_dir%/../vendor/friendsofvictoire',
```

Comment:
If cache:warmup breaks, it is always possible to support some "exclusion" in the "base_paths" like `!%kernel.root_dir%/../vendor/victoire/victoire/Tests`.
It would be necessary to support this exclusion in the annotationDriver:getAllClassNames() method